src/main/java/org/sdnplatform/sync/internal/SyncManager.java

```diff
@@ -812,4 +815,12 @@ private SyncMessage getMessage(SynchronizingStorageEngine store) {
 			return bsm;
 		}
 	}
+
+	@Override
```

Comment:
@tulioalberton Ah, this is what you were talking about earlier with the RPCService and needing a listener, right? Right now, if I understand correctly, you're polling for the connections periodically and would like to have a way for the RPCService to notify you upon a connection add/remove event.